ambry-store/src/main/java/com.github.ambry.store/DiskManager.java

```diff
@@ -239,6 +239,21 @@ boolean disableCompactionForBlobStore(PartitionId id) {
     return store != null && compactionManager.disableCompactionForBlobStore(store);
   }
 
+  /**
+   * Shutdown the {@link PartitionId} {@code id}.
+   * @param id the {@link PartitionId} of the {@link BlobStore} which should be shutdown.
+   */
+  void shutdownBlobStore(PartitionId id) {
```

Comment:
shouldn't we return `boolean` here? so that we can return an error if the stop store failed?